src/button-split.scss

```diff
@@ -76,4 +76,15 @@ $fd-button-split-margin: 0.125rem / 2;
       }
     }
   }
+
+  .fd-button__text {
```

Comment:
Let's add `fd-namespace` instead of `fd-`

Reply:
fixed